<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="ConfigTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item row="10" column="2">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>CH3 Spec (Celcius):</string>
          </property>
         </widget>
        </item>
        <item row="17" column="2">
         <widget class="QCheckBox" name="checkBoxCH3">
          <property name="text">
           <string>CH3</string>
          </property>
         </widget>
        </item>
        <item row="10" column="3">
         <widget class="QLineEdit" name="lineEditCH3Spec">
          <property name="text">
           <string>-50</string>
          </property>
         </widget>
        </item>
        <item row="15" column="2">
         <widget class="QCheckBox" name="checkBoxAlarmEmail">
          <property name="text">
           <string>Alert Email</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QCheckBox" name="checkBoxCH2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>CH2</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Flipper addr:</string>
          </property>
         </widget>
        </item>
        <item row="29" column="0" colspan="2">
         <widget class="QPushButton" name="pushButtonStart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Flipper IP:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>CH1 Spec (Celcius):</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QCheckBox" name="checkBoxStatusMobile">
          <property name="text">
           <string>Status Mobile</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>CH2 Spec (Celcius):</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Warehouse Name:</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QCheckBox" name="checkBoxStatusEmail">
          <property name="text">
           <string>Status Email</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEditWarehouseName"/>
        </item>
        <item row="17" column="0">
         <widget class="QCheckBox" name="checkBoxCH1">
          <property name="text">
           <string>CH1</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QLineEdit" name="lineEditFlipperPort">
          <property name="text">
           <string>502</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEditFlipperIP"/>
        </item>
        <item row="11" column="1">
         <widget class="QLineEdit" name="lineEditCH2Spec">
          <property name="text">
           <string>-50</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="label_2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Port:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLineEdit" name="lineEditFlipperAddr">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="15" column="3">
         <widget class="QCheckBox" name="checkBoxAlarmMobile">
          <property name="text">
           <string>Alert Mobile</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="lineEditCH1Spec">
          <property name="text">
           <string>-50</string>
          </property>
          <property name="maxLength">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelLocalIP">
          <property name="text">
           <string>Local IP:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Raspberry Pi addresses:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="4">
         <widget class="Line" name="line_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="4">
         <widget class="Line" name="line_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="4">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="16" column="0" colspan="4">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="19" column="0" colspan="4">
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="29" column="2">
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item row="29" column="3">
         <widget class="QPushButton" name="buttonResetDatabase">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Reset Database</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLabel" name="labelPublicIP">
          <property name="maximumSize">
           <size>
            <width>410</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Public IP:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Decimalpoint (10/100/1000): </string>
          </property>
         </widget>
        </item>
        <item row="11" column="3">
         <widget class="QLineEdit" name="lineEditDecimalpoint"/>
        </item>
        <item row="17" column="3">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CH1">
       <attribute name="title">
        <string>CH1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCustomPlot" name="GraphCH1" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CH2">
       <attribute name="title">
        <string>CH2</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="CH3">
       <attribute name="title">
        <string>CH3</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
